tests/lib/managers/webhooks-test.js

```diff
@@ -153,6 +153,26 @@ describe('Webhooks', function() {
 
 	});
 
+	describe('setSignatureKeys()', function() {
+		const PRIMARY_SIGNATURE_KEY = 'SamplePrimaryKey';
+		const SECONDARY_SIGNATURE_KEY = 'SampleSecondaryKey';
+
+		it('should validate the primary signature key is set', function() {
+			Webhooks.setSignatureKeys(PRIMARY_SIGNATURE_KEY, SECONDARY_SIGNATURE_KEY);
+
+			assert.equal(PRIMARY_SIGNATURE_KEY, Webhooks.primarySignatureKey);
+			assert.equal(SECONDARY_SIGNATURE_KEY, Webhooks.secondarySignatureKey);
+		});
+
+		it('should validate the secondary signature key is undefined', function() {
```

Comment:
This test name isn't very descriptive; maybe something like "should set the primary key when no secondary key is passed"?